build-tools/installers/create-installers.targets

```diff
@@ -0,0 +1,113 @@
+    <MSBuildFiles Include="$(MSBuildSrcDir)\jcw-gen.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\jnimarshalmethod-gen.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\LayoutBinding.cs"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\libZipSharp.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\libZipSharp.dll.config"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\logcat-parse.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\mdoc.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\mkbundle.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\mkbundle-api.h"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\mono-symbolicate.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.CSharp.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Options.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Terminal.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\monodoc.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\MULTIDEX_JAR_LICENSE"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\**\proguard.jar"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\**\license.html"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\r8.jar"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\SgmlReaderDll.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin*"></MSBuildFiles>
+  </ItemGroup>
+  <ItemGroup> <!-- monodroid -->
+    <MSBuildFiles Condition="Exists('$(MSBuildSrcDir)\ICSharpCode.SharpZipLib.dll')" Include="$(MSBuildSrcDir)\ICSharpCode.SharpZipLib.dll"></MSBuildFiles>
+    <MSBuildFiles Condition="Exists('$(MSBuildSrcDir)\INIFileParser.dll')" Include="$(MSBuildSrcDir)\INIFileParser.dll"></MSBuildFiles>
+    <MSBuildFiles Condition="Exists('$(MSBuildSrcDir)\jar2xml.jar')" Include="$(MSBuildSrcDir)\jar2xml.jar"></MSBuildFiles>
+    <MSBuildFiles Condition="Exists('$(MSBuildSrcDir)\Mono.AndroidTools.dll')" Include="$(MSBuildSrcDir)\Mono.AndroidTools.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Android.DebugRuntime-*"/>
+  </ItemGroup>
+  <ItemGroup>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\cross-arm.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\cross-arm64.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\cross-x86.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\cross-x86_64.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\opt.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\llc.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\**\proguard.bat"></MSBuildFilesWin>
+  </ItemGroup>
+  <ItemGroup>
+    <MSBuildFilesDarwin Include="$(MSBuildSrcDir)\**\cross-arm"></MSBuildFilesDarwin>
```

Comment:
This set should be `@(MSBuildFilesUnix)`, as Unix should also provide these files.